src/pagecache/mod.rs

```diff
@@ -59,16 +59,19 @@ pub use self::{
     segment::SegmentMode,
 };
 
-/// An offset for a storage file segment.
+/// A segment id == Segment.LogOffset / Config.SegmentSize.
```

Comment:
I updated it as you asked. I think segment offset instead of Id is a little bit confusing because all offsets are used in relation to the database log. And the factual segment offset is expressed with LSN. Thus we are having 2 segment offsets by definitions.